docs/customization/configuration.md

```diff
@@ -0,0 +1,129 @@
+# Configuring Your App
+
+## `settings_local.py`
+
+Unlike a vanilla django installation, our `onisite/settings.py` is not where
+all settings are stored. This file actually combines django's defaults
+(`onisite/django_defaults.py`), the Open ONI defaults
+(`onisite/settings_base.py`), and your local configuration
+(`onisite/settings_local.py`).
+
+Basic site customizations can be done by creating `onisite/settings_local.py`
+in the same location you find `onisite/settings_base.py`, and overriding
+values. The easiest option is to copy the example file:
+
+```bash
+cp onisite/settings_local_example.py onisite/settings_local.py
+```
+
+Then adjust settings, particularly those starting with `YOUR`.
+
+There are many settings in the example file which are based on environment
+variables and/or other settings. Only change these if you understand the
+ramifications of doing so. Most users will be fine just letting those stay
+as-is.
+
+That said, overrides are simple - you just redefine a value. For example, you
+could do a simple change to enable TIFFs by putting this line into your
+`onisite/settings_local.py`:
+
+```python
+USE_TIFF = True
```

Comment:
I don't like giving an indication of enabling TIFFs in our basic documentation.  That feature is a terrible idea for 95% of users, and I'm not sure if it even works at this point - I certainly haven't tested it in many years.

Maybe there's a "safer" setting to point out as an example instead?  I like `THUMBNAIL_WIDTH` since it's not something most people need, but it is relevant and useful to customize in a handful of situations.

Reply:
Sounds good!

Reply:
The USE_TIFF line caught my attention when I initially reviewed things but I was on the fence about mentioning it. I think Jeremy's right though. Not something we should highlight :)